lib/common/issues/src/dashboard.rs

```diff
@@ -7,22 +9,32 @@ use crate::issue::{CodeType, Issue, IssueRecord};
 #[derive(Default)]
 struct Dashboard {
     pub issues: DashMap<CodeType, IssueRecord>,
+    inverted_index: DashMap<TypeId, HashSet<CodeType>>,
```

Comment:
I think this inverted index is a bit overkill given that there will be often a very low number of issues.

Reply:
How about using a composite `Code` type instead of a plain string (https://github.com/qdrant/qdrant/pull/4086/commits/3827ebbcb47bad353abb6423d6e18ebec2d41255)?

Reply:
I like the structured `Code` better :+1: